micro_dl/utils/masks.py

```diff
@@ -94,3 +95,65 @@ def create_unimodal_mask(input_image, str_elem_size=3):
     thr_image = binary_opening(input_image > thr, str_elem)
     mask = binary_erosion(thr_image, str_elem)
     return mask
+
+
+def get_unet_border_weight_map(annotation, w0=10, sigma=5):
+    """
+    Generate the weight map for borders as specified in the UNet paper for an annotation.
+    Parameters
```

Comment:
Could you please make the docstrings similar to the ones we have in the repo? On hindsight numpy style would've been better...